### PR TITLE
Feature/implement eslint

### PR DESCRIPTION
**Pull request by miayang0513**
- Use [Webpack](https://webpack.js.org/) to package and build project
- Apply [Standard JS](https://standardjs.com/) as team coding style and use [ESlint](https://eslint.org/) to force developer comply with it

### CODE REVIEW

**Comment by tsungtingdu** [comment]
Link issue #4

**Comment by tsungtingdu** [comment]
@miayang0513 test slack message

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  extends: [
+    'standard'
+  ],
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: 'module'
+  },
+  rules: {
+  }
+}
```
Comment (tsungtingdu) [inline .eslintrc.js:16]: 這裡可以加個
```
 ignorePatterns: ['**/dist'],
```
來 skip 掉 dist 嗎？

**Review by tsungtingdu — COMMENTED** [review]
@miayang0513 LGTM! only few questions here

File: package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "ac-ta-tool-helper",
+  "version": "1.0.0",
+  "description": "This is a Chrome extension that supports TA's work.",
+  "main": "main.js",
+  "scripts": {
+    "dev": "webpack --config webpack/webpack.dev.js --watch",
+    "build": "webpack --config webpack/webpack.prod.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
[the remark this answers is not in the document]
Comment (miayang0513) [reply package.json:10]: @tsungtingdu 有需要嗎？因為 npm run dev 之後，只要按下 save 都會自動偵測，不過要加也可以

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,2890 @@
+{
+  "name": "ac-ta-tool-helper",
+  "version": "1.0.0",
+  "lockfileVersion": 1,
```
[the remark this answers is not in the document]
Comment (miayang0513) [reply package-lock.json:4]: @tsungtingdu 這感覺要跟 CHANGELOG 一致

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (miayang0513) [reply package.json:10]: @tsungtingdu 嗯嗯 沒關係 我就加上吧

File: package-lock.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (miayang0513) [reply package-lock.json:4]: okk

[the remark this answers is not in the document]
Comment (miayang0513) [reply package-lock.json:4]: @tsungtingdu 所以 lockfileVersion 我要 update 成 2 嗎？

File: README.md [hunk 4]
```diff
@@ -16,5 +16,23 @@ This is a Chrome extension that supports TA's work.
...
 4. You can find "AC TA Tool Helper" when right clicking the page
+
+## Development Setup
+```
+# watch and auto re-build dist folder
+$ npm run dev
+
+# build dist folder in production
+$ npm run build
+```
```
[the remark this answers is not in the document]
Comment (miayang0513) [reply README.md:29]: ok

File: .eslintrc.js (hunk 1 above)
Comment (miayang0513) [reply .eslintrc.js:16]: sure

File: package-lock.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (tsungtingdu) [reply package-lock.json:4]: let's skip this for now :)

**Review by tsungtingdu — APPROVED** [review]
@miayang0513 well done! ready for merge

**Pull request merged**